labtool.py

```diff
@@ -574,6 +592,8 @@ def execute_stages(path, stage=0):
         logger.info("applying stage %s configuration" % s)
         logger.debug("opening stage commands in %s" % fname)
         try:
```

Comment:
filename for stage is still limited to 'stage_#' and the number of stages are determined by scanning the lab directory.  Do we need to change this behavior or relax the check to do 'stage_#_<arbitrary string>'?  Right now I don't think your stages '01_LLDP.sh' will be found or executed